src/components/ComputeModal.js

```diff
@@ -1126,6 +1136,13 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
             ]),
             h(SaveFilesHelp)
           ])],
+          [willDeletePersistentDiskForRstudio(), () => h(Fragment, [
```

Comment:
And this could be:
```
[willDeletePersistentDisk(), () => h(Fragment, [
            p([
              'Reducing the size of a persistent disk requires it to be deleted and recreated. This will ',
              span({ style: { fontWeight: 600 } }, ['delete all files on the disk.'])
            ]),
            isToolRStudio() ? h(SaveFilesHelp) : h(SaveFilesHelpRStudi)
          ])],
```

Reply:
I was initially thinking the whole text will look a bit different..but I think we end up only changing the text in `SaveFileHelp` part..so we should be able to remove `willDeletePersistentDiskForRStudio`